lib/Cake/Model/Behavior/TreeBehavior.php

```diff
@@ -997,6 +997,41 @@ public function verify(Model $Model) {
 	}
 
 /**
+ * Returns the depth level of a node in the tree.
+ *
+ * @param Model $Model Model using this behavior
+ * @param int|string $id The primary key for record to get the level of.
+ * @return int|bool Integer of the level or false if the node does not exist.
+ */
+	public function getLevel(Model $Model, $id = null) {
+		if ($id === null) {
+			$id = $Model->id;
+		}
+
+		$node = $Model->find('first', array(
+			'conditions' => array($Model->escapeField() => $id),
+			'order' => false,
+			'recursive' => -1
+		));
+
+		if (empty($node)) {
+			return false;
+		}
+
+		extract($this->settings[$Model->alias]);
+
+		return $Model->find('count', array(
+			'conditions' => array(
+				$scope,
```

Comment:
@markstory It can be. Not that it matters. The scope is merged with conditions the same way in various other methods.